src/nidigital/examples/burst_with_start_trigger/nidigital_burst_with_start_trigger_grpc.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import nidigital
+import os
+import sys
+
+
+def example(resource_name, options, address, port, trigger_source=None, trigger_edge=None):
```

Comment:
You'll need to add address and port here as parameters since you're now passing them in from main()

Reply:
Clarity I would just prefer `def example(resource_name, options, trigger_source=None, trigger_edge=None, address, port):` if possible to match other examples

Reply:
Based on a previous conversation, it errors when in the end so I placed the parameters back in the beginning

Reply:
Great thank you